src/gburtini/Distributions/Binomial.php

```diff
@@ -106,11 +107,9 @@ public function icdf($p)
         } while ($k < $this->n);
         return $k;
     }
-}
-
 
-class BinomialCI extends Binomial
-{
+    // there is no reason why BinomialCi is created, it can be method of Binomial
```

Comment:
Agreed. This should have been a comment on the PR rather than a comment in the code though.